src/fixtures/products.js

```diff
@@ -0,0 +1,63 @@
+export const products = [
+  {
+    id: "76w0hz7015kkr9kjkav",
+    images: [
+      "https://content2.rozetka.com.ua/goods/images/big_tile/163399632.jpg",
+    ],
+    title: "Ноутбук Acer Aspire 3 A315-57G-336G (NX.HZREU.01S) Charcoal Black",
+    rating: 2.89,
+    price: 100,
+    category: "laptops",
+    brand: "acer",
+    count: 1,
+  },
+  {
+    id: "qeagrlm9lrkr9kjkav",
+    images: [
+      "https://content1.rozetka.com.ua/goods/images/big_tile/178060622.jpg",
+    ],
+    title: "Ноутбук Acer Aspire 7 A715-41G-R9KP (NH.Q8QEU.00L) Charcoal Black",
+    rating: 1.96,
+    price: 200,
+    category: "laptops",
+    brand: "acer",
+    count: 1,
+  },
+  {
+    id: "0y9ksratv6akr9kjkav",
+    images: [
+      "https://content2.rozetka.com.ua/goods/images/big_tile/178060660.jpg",
+    ],
+    title: "Ноутбук Acer Aspire 7 A715-75G-51ZW (NH.Q88EU.00P) Charcoal Black",
+    rating: 2.42,
+    price: 300,
+    category: "laptops",
+    brand: "acer",
+    count: 1,
+  },
+  {
+    id: "cvr29caokhhkr9kjkav",
+    images: [
+      "https://content1.rozetka.com.ua/goods/images/big_tile/25101152.jpg",
+    ],
+    title:
+      "Ноутбук Acer Nitro 5 AN515-55-56WH (NH.Q7PEU.00L) Obsidian Black Суперцена!!!",
+    rating: 0.53,
+    price: 400,
+    category: "laptops",
+    brand: "acer",
+    count: 1,
+  },
+  {
+    id: "k9hb29sfeekr9kjkav",
+    images: [
+      "https://content1.rozetka.com.ua/goods/images/big_tile/24790127.jpg",
+    ],
+    title: "Ноутбук Acer Aspire 7 A715-75G-57LR (NH.Q87EU.006) Charcoal Black",
+    rating: 3.2,
+    price: 500,
+    category: "laptops",
+    brand: "acer",
+    count: 1,
+  },
+];
```

Comment:
**Performance Issue**: The static data for products is directly embedded within the JavaScript file, which can lead to increased load times and reduced performance as the size of the data grows. <br> **Fix**: Move the product data to a separate JSON file and fetch it asynchronously to improve load times and scalability. <br> **Code Suggestion**: 
 ```
 -export const products = [
 -  {
 -    id: "76w0hz7015kkr9kjkav",
 -    images: [
 -      "https://content2.rozetka.com.ua/goods/images/big_tile/163399632.jpg",
 -    ],
 -    title: "Ноутбук Acer Aspire 3 A315-57G-336G (NX.HZREU.01S) Charcoal Black",
 -    rating: 2.89,
 -    price: 100,
 -    category: "laptops",
 -    brand: "acer",
 -    count: 1,
 -  },
 -  {
 -    id: "qeagrlm9lrkr9kjkav",
 -    images: [
 -      "https://content1.rozetka.com.ua/goods/images/big_tile/178060622.jpg",
 -    ],
 -    title: "Ноутбук Acer Aspire 7 A715-41G-R9KP (NH.Q8QEU.00L) Charcoal Black",
 -    rating: 1.96,
 -    price: 200,
 -    category: "laptops",
 -    brand: "acer",
 -    count: 1,
 -  },
 -  {
 -    id: "0y9ksratv6akr9kjkav",
 -    images: [
 -      "https://content2.rozetka.com.ua/goods/images/big_tile/178060660.jpg",
 -    ],
 -    title: "Ноутбук Acer Aspire 7 A715-75G-51ZW (NH.Q88EU.00P) Charcoal Black",
 -    rating: 2.42,
 -    price: 300,
 -    category: "laptops",
 -    brand: "acer",
 -    count: 1,
 -  },
 -  {
 -    id: "cvr29caokhhkr9kjkav",
 -    images: [
 -      "https://content1.rozetka.com.ua/goods/images/big_tile/25101152.jpg",
 -    ],
 -    title: "Ноутбук Acer Nitro 5 AN515-55-56WH (NH.Q7PEU.00L) Obsidian Black Суперцена!!!",
 -    rating: 0.53,
 -    price: 400,
 -    category: "laptops",
 -    brand: "acer",
 -    count: 1,
 -  },
 -  {
 -    id: "k9hb29sfeekr9kjkav",
 -    images: [
 -      "https://content1.rozetka.com.ua/goods/images/big_tile/24790127.jpg",
 -    ],
 -    title: "Ноутбук Acer Aspire 7 A715-75G-57LR (NH.Q87EU.006) Charcoal Black",
 -    rating: 3.2,
 -    price: 500,
 -    category: "laptops",
 -    brand: "acer",
 -    count: 1,
 -  },
 -];
 +import products from './products.json';
 ```